src/Fuel-Platform-Core/FLExtensionWrapper.class.st

```diff
@@ -19,12 +19,10 @@ FLExtensionWrapper class >> fromPragma: aPragma [
 
 { #category : #private }
 FLExtensionWrapper >> compileInTarget [
-	| methodSelector source |
-	source := self method methodClass instanceSide
+	| source |
+	source := self method methodClass theNonMetaClass
```

Comment:
@theseion I think #theNonMetaClass will be deprecated. Maybe  removing #methodSelector and using #instanceSide were changes not ported to Fuel repository?